src/actions/student.js

```diff
@@ -1,14 +1,14 @@
-export const STUDENT_FETCH_REQUESTED = 'STUDENT_FETCH_REQUESTED';
-export const STUDENT_FETCH_SUCCEDED = 'STUDENT_FETCH_SUCCEDED';
-export const STUDENT_FETCH_FAILED = 'STUDENT_FETCH_FAILED';
+export const STUDENT_FETCH_REQUESTED = 'STATS_FETCH_REQUESTED';
+export const STUDENT_FETCH_SUCCEEDED = 'STATS_FETCH_SUCCEEDED';
+export const STUDENT_FETCH_FAILED = 'STATS_FETCH_FAILED';
```

Comment:
I called it that for the searching of a student and adding of a student and not for getting the stats of a student that is already within a program.